utils/spacewalk-common-channels

```diff
@@ -106,6 +117,16 @@ def add_channels(channels, section, arch):
             channel[k] = config.get(section, k)
         channels[channel['label']] = channel
 
+
+def channel_exists(client, base_channel_label):
+    # check whether channel exists
+    try:
+        base_info = client.channel.software.getDetails(key, base_channel_label)
+        return base_info
+    except xmlrpclib.Fault, e:
+        return False
```

Comment:
I would return `None`.

You either get the structure back or `None`. The "if" statement works also with None.